projects/NiftySubs.md

```diff
@@ -0,0 +1,98 @@
+Under Development
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
+### Home Page
+![image](https://github.com/NiftySubs/niftysubs/blob/master/src/assets/HomePage.png)
+
+### Create Stream
+![image](https://github.com/NiftySubs/niftysubs/blob/master/src/assets/Create%20Stream.png)
+
+### Fundraisers
+![image](https://github.com/NiftySubs/niftysubs/blob/master/src/assets/Fundraisers.png)
+
+### Superchats
+![image](https://github.com/NiftySubs/niftysubs/blob/master/src/assets/Superchats.png)
+
+## Target Audience
+<!--Describe who will be your project's users-->
+- Gamers
+- Sports Channels
+- People who want to build communities.
+
+## Rough estimated user base (if applicable)
+<!--How many users do you have right now?-->
+¯\_(ツ)_/¯
+
+## Github repo
+<!--Attach a link to your GitHub repo - open source is required - please make sure your repo has a license file and is licensed using MIT open source license! -->
+[Github Repo](https://github.com/NiftySubs/niftysubs)
+
+## Website
+<!--Link your website if available-->
+  [NiftySubs Website](https://niftysubs.com)
+
+<!--If you're applying for a Next Step grant, add the URL to your hackathon submission here also-->
+### Hackathon Submission
+  [HackMoney 2021](https://showcase.ethglobal.co/hackmoney2021/niftysubs)
+
+## Docs
+<!--Including a link to your project docs!-->
+[Decentralized Livestream Platform](https://github.com/NiftySubs/niftysubs)
+
+## Team Info
+<!-- Introduce your amazing team - how many team members are working on this project and who are they?-->
+
+### Team Size 
+ 5
+
+### Team members
+  #### Harpalsinh Jadeja
+  Harpal is from Mumbai, India. Recent graduate in Bachelors Of Engineering in Computer Engineering. Harpal works at L & T Infotech as a Oracle Developer. Harpal is enthusiastic
+  about crypto, more on the development side of it. Harpal's passion to bring web2 users to the web3.
+  
+  #### Sushant Kumar
+  Sushant was one of the co-founders of the startup Azuro that got acquired by SquareYards. 
+  Sushant love building challenging stuff. Sushant is a very passionate about coding and Sushant can hack away for days at problems.
+  Sushant has been playing around with building some ERC20 tokens and some decentralised apps on top of BitClout. Sushant is fluent with Javascript (ReactJS), Python (Flask, FastAPI), Solidity, HTML and CSS. 
+  Sushant is an enthusiastic builder and would love to be a part of any team that shares that enthusiasm.
+  
+  #### Elio Jordan Lopes
+  Elio Jordan Lopes is interested in Full stack web/mobile, web3, developer and is a student. 
+  
+  #### Huy Vu Nguyen
+  Product and Design in the web3 ecosystem
+  
+  #### Rishabh Keshan
+  Mern Stack Developer, React Native, Web3, UI/UX Developer
+
+
+## How the community can engage
+* GitHub Discussion: <!--Start a discussion with the community here: https://github.com/ipfs/community/discussions/new and attach the link!-->  
```

Comment:
Please create a discussion thread and link it here

Reply:
Github Discussion Link Added!